ppr-ui/src/views/mhrInformation/MhrInformation.vue

```diff
@@ -892,6 +892,9 @@ export default defineComponent({
         'Ownership Transfer or Change'
       ),
       isChangeOwnershipBtnDisabled: computed((): boolean => {
+        if(isRoleStaffReg.value && hasLien.value)
```

Comment:
This may not work totally as intended, as this will ignore the rest of the conditions within the computed value. 
They may not all be applicable to staff but i think if an MHR is FROZEN and due to AFFE we would want the button to be disabled, if we add this it will be enabled if there is also a lien registered against the MHR.

Can either modify the functions return or include the !isFrozenMhrDueToAffidavit.value in your condition here. 

Also would use probably use { } if putting the return on a new line, generally.

Reply:
@cameron-eyds  addressed your comment.

Reply:
Thank you!